src/main/scala/scalaz/ziokeeper/VectorClock.scala

```diff
@@ -0,0 +1,63 @@
+package scalaz.ziokeeper
+
+import scala.annotation.tailrec
+import VectorClock._
+case class VectorClock[A](value: Map[A, Long]) extends AnyVal {
```

Comment:
This is great for now, ultimately, `Map` won't be an efficient way to store or represent a vector clock. 👍